packages/store-ui/src/atoms/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import Checkbox from './Checkbox'
+
+describe('Checkbox', () => {
+  it('data-store-checkbox is present', () => {
+    const { getByTestId } = render(<Checkbox testId="checkbox" />)
```

Comment:
```suggestion
    const { getByTestId } = render(<Checkbox />)
```

Reply:
Passing the testId, instead of using the default one, prevents break the test furthermore, what do you think?

Reply:
You have a point. Maybe really better pass the testid.